conans/client/manager.py

```diff
@@ -95,7 +103,8 @@ def package_files(self, reference, package_folder, profile, force):
         remote_proxy = ConanProxy(self._client_cache, self._user_io, self._remote_manager,
                                   remote_name=None, update=False, check_updates=False,
                                   manifest_manager=None)
-        loader = ConanFileLoader(self._runner, self._client_cache.settings, profile)
+
+        loader = ConanFileLoader(self._runner, self._client_cache.settings, self._profile_with_defaults(profile))
```

Comment:
I am not sure about this, or at least I don't understand it. 
Previously, the ``profile`` was being used, but it wasn't the default one, it could be one defined by ``--profile=myprofile``. Now it seems that the default is always applied, that changes behavior.